resources/roadmap/index.md

```diff
@@ -221,16 +235,13 @@ meta_descr: meta_descr.roadmap
                         <h2>{% t roadmap.future %}</h2>
                             <ul>
                                 <h3 class="months">{% t roadmap.comingsoon %}</h3>
-                                    <li class="ongoing"><a href="https://haveno.exchange">{% t roadmap.haveno %}</a></li>
-                                    <li class="ongoing"><a href="https://github.com/seraphis-migration/strategy/wiki">{% t roadmap.seraphis-jamtis %}</a></li>
                                     <li class="ongoing"><a href="https://github.com/Rucknium/OSPEAD">{% t roadmap.ospead %}</a></li>
-                                    <li class="ongoing">{% t roadmap.ringsize-sera-tryp %}</li>
-                                    <li class="ongoing">{% t roadmap.tryptych %}</li>
                                     <li class="ongoing">{% t roadmap.xmr-eth-atomicswaps %}</li>
                                     <li class="ongoing">{% t roadmap.bp++ %}</li>
-                                    <li class="ongoing">{% t roadmap.lws-subaddress %}</li>
-                                    <li class="upcoming">{% t roadmap.returnaddr %}</li>
-                                    <li class="upcoming">{% t roadmap.layer2 %}</li>
+                                    <li class="ongoing">{% t roadmap.cuprate %}</li>
+                                    <li class="upcoming"><a href="https://github.com/seraphis-migration/strategy/wiki">{% t roadmap.seraphis-jamtis %}</a></li>
+                                    <li class="upcoming"><a href="https://www.getmonero.org/2024/04/27/fcmps.html">{% t roadmap.fcmp-plus %}</a></li>
```

Comment:
this one

Reply:
Can you show me what you want to change/remove/keep? its not clear to me

Reply:
move ``<a href="https://github.com/seraphis-migration/strategy/wiki">`` line 243 from line 242

Reply:
make these changes as @nahuhh seems to agree and we can review further. What will `Seraphis codebase and jamtis` link to then?